File: backend/components/metiers/threed/legacy/documentation/checklists/checklist_generator.js
// Script JS pour générer automatiquement des checklists de documentation legacy Threed
// Conforme au cahier des charges, extensible, prêt à l’emploi
function generateChecklist(items) {
  console.log('Checklist générée :');
  items.forEach(item => console.log(`- [ ] ${item}`));
}

if (require.main === module) {
  const items = [
    'Documentation technique complète',
    'Documentation fonctionnelle complète',
    'Modèles de guides prêts à l’emploi',
    'Changelog documenté',
    'Accessibilité et multilingue'
  ];
  generateChecklist(items);
}

File: backend/components/metiers/threed/legacy/documentation/checklists/checklist_generator.py
"""
Script Python pour générer automatiquement des checklists de documentation
legacy Threed.
Conforme au cahier des charges, extensible, prêt à l’emploi.
"""


def generate_checklist(items):
    print("Checklist générée :")
    for item in items:
        print(f"- [ ] {item}")


if __name__ == "__main__":
    items = [
        "Documentation technique complète",
        "Documentation fonctionnelle complète",
        "Modèles de guides prêts à l’emploi",
        "Changelog documenté",
        "Accessibilité et multilingue",
    ]
    generate_checklist(items)
